README.md

```diff
@@ -1,26 +1,341 @@
+| `yaml`     | `changelog.yaml`       | Path to the changelog.yaml file                                                                                                |
+| `markdown` | `CHANGELOG.partial.md` | Path to the destination markdown file                                                                                          |
+| `version`  |                        | Version to stamp in the changelog section header. If omitted, no version header will be generated                              |
+| `date`     | `time.Now()`           | Date to stamp in the changelog section header, in YYYY-MM-DD format. If empty it will default to the current time (time.Now()) |                                                                                                                                                                                                          |
+
+### Update markdown
+Incorporates a changelog.yaml into a complete CHANGELOG.md.
+```shell
+rt update-markdown [-flags]
+```
+| Flags      | Default          | Description                                                                                                                    |
+|------------|------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| `yaml`     | `changelog.yaml` | Path to the changelog.yaml file                                                                                                |
+| `markdown` | `CHANGELOG.md`   | Path to the destination markdown file                                                                                          |
+| `version`  |                  | Version to stamp in the changelog section header. If omitted, no version header will be generated                              |
+| `date`     | `time.Now()`     | Date to stamp in the changelog section header, in YYYY-MM-DD format. If empty it will default to the current time (time.Now()) |                                                                                                                                                                                                          |
+
+### Validate markdown
+Prints errors if CHANGELOG.md has an invalid format.
+```shell
+rt validate-markdown [-flags]
+```
+| Flags           | Default        | Description                       |
+|-----------------|----------------|-----------------------------------|
+| `markdown`      | `CHANGELOG.md` | Validate specified changelog file |
+| `exit-code`     | `1`            | Exit code when errors are found   |
+
+## Actions
+
+- [Generate YAML changelog](./generate-yaml/README.md)
+- [Is Held](./is-held/README.md)
+- [Link dependencies](./link-dependencies/README.md)
+- [Next Version](./next-version/README.md)
+- [render](./render/README.md)
+- [Update markdown](./update-markdown/README.md)
+- [Validate markdown](./validate-markdown/README.md)
```

Comment:
Will these links work regardless of the page where they're being visited?

I think that if they are on the front page, they will redirect to `https://github.com/newrelic/release-toolkit/generate-yaml/README.md`, which does not exist. The correct URL needs to include `/blob/main/`.

Including `blob/main` unconditionally, however, will not work if you are viewing the README from its dedicated page: https://github.com/newrelic/release-toolkit/blob/main/README.md.